### PR TITLE
Add Discord Webhook notification

### PR DESCRIPTION
**Pull request by denverquane**
Adds functionality to invoke a Discord webhook and post to a channel whenever a campsite is found

The image below is what a user would see in their Discord channel if a campsite is found (this image shows 0 sites purely because of testing; *the bot does not post Discord messages at all if no sites are found*)

Untitled

### CODE REVIEW

**Comment by banool** [comment]
Thanks for the PR! There have been other PRs throughout the last few years where people add new notification methods. As with those, I request the same here, could you refactor such that there is a directory like `notifiers/` and this is just one of the notifiers under there. Ideally we also have feature gating around which notifiers you use, including of dependencies. This latter point is a nice to have though, given that's easier said than done in Python.

In short, if you could pull out the discord stuff from the main code wherever possible I'd appreciate it!

**Comment by banool** [comment]
Closing this since the requested changes weren't made. Feel free to reopen with updates!
